src/ol/renderer/webgl/map.js

```diff
@@ -465,6 +463,8 @@ ol.renderer.webgl.Map.prototype.renderFrame = function(frameState) {
   gl.enable(ol.webgl.BLEND);
   gl.viewport(0, 0, this.canvas_.width, this.canvas_.height);
 
+  this.dispatchComposeEvent_(ol.render.EventType.PRECOMPOSE, frameState);
```

Comment:
Mimicking the class "ol.renderer.canvas.Map.prototype.renderFrame", the render event "PRECOMPOSE" is fired after the "gl.clearColor()" statement so that I have a good chance of customizing the map background color in WebGL mode. Otherwise, my custom background color is always reset by "gl.clearColor(0, 0, 0, 0);".